recipes/rsgislib/bld.bat

```diff
@@ -0,0 +1,41 @@
+
+%LIBRARY_BIN%\cmake -G "NMake Makefiles" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
```

Comment:
Yes but I get a CMAKE_ROOT not found error if I don't do this. I'll investigate...